.github/workflows/release.yaml

```
@@ -260,9 +259,40 @@ jobs:
       run: |
         gh release create $RELEASE_VERSION -t "OpenShift Service Mesh Console $RELEASE_VERSION"
 
-    - name: Create or update version branch
+    - name: Check if version branch exists
+      if: ${{ needs.initialize.outputs.release_type == 'patch' }}
+      run: |
+        if git ls-remote --exit-code --heads origin "$BRANCH_VERSION"; then
+          echo "EXISTS=true" >> $GITHUB_ENV
+        else
+          echo "EXISTS=false" >> $GITHUB_ENV
+        fi
+
+    - name: Create version branch
+      if: ${{ env.EXISTS == 'false' && needs.initialize.outputs.release_type == 'patch' }}
       run: git push origin $(git rev-parse HEAD):refs/heads/$BRANCH_VERSION
```

Do we have a use case when the release type is `patch` and the the version branch is not created? If the answer is no, I prefer to throw an error because this means something wrong is happening.

---

I added this check for this PR only, thought it was an improvement, but really, when branch doesn't exist, it means is the first time creating the version branch, meaning, a minor release. 

So, both cases are going to happen, when false, it's a minor, and the branch needs to be created.

When true, it already exists, it's a patch release. I'm going to remove the and condition that checks the release type, because the false is already enought I think.

---

Well... I changed the approach a little bit, you'll notice that:

1. if it is a minor/major release, branch won't exist, needs to be created
2. if it is a patch release, create a PR directly